### PR TITLE
Port System.Linq.Expressions source comments to Docs

### PR DESCRIPTION
**Pull request by carlossanlop**
## Summary

Automatically porting a few triple slash source code comments found in System.Linq.Expressions that were not yet documented in dotnet-api-docs.

Fixes #Issue_Number (if available)

### CODE REVIEW

**Comment by carlossanlop** [comment]
@mairaw @rpetrusha please take a look whenever possible.

Adding the area owners @cston, @333fred. Charles, Fred, can you please take a look? The summary messages I ported are all generic and have exactly the same string, so it would be awesome if the code experts could jump in to edit them into some more descriptive and specific summaries.

**Comment by carlossanlop** [comment]
@mairaw @rpetrusha if this looks good enough, can we get it merged?

**Comment by carlossanlop** [comment]
> The summaries look good, @carlossanlop, but the parameter descriptions and return values are missing. Would it be possible for you to also add those? The parameter description is, "The visitor to visit this node with." The return value description is "The result of visiting this node."

For the visitor parameter, would it sound better to use the following instead?:

"The &lt;see cref="T:System.Linq.Expressions.ExpressionVisitor"&gt; with which this node will be visited."

File: xml/System.Linq.Expressions/DefaultExpression.xml [hunk 1]
```diff
@@ -81,7 +81,7 @@
       </Parameters>
       <Docs>
         <param name="visitor">To be added.</param>
```
Comment (carlossanlop) [inline xml/System.Linq.Expressions/DefaultExpression.xml]:
```suggestion
        <param name="visitor">The <see cref="T:System.Linq.Expressions.ExpressionVisitor"> with which this node will be visited.</param>
```

File: xml/System.Linq.Expressions/Expression`1.xml [hunk 2]
```diff
@@ -101,7 +101,7 @@
       </Parameters>
       <Docs>
         <param name="visitor">To be added.</param>
```
Comment (carlossanlop) [inline xml/System.Linq.Expressions/Expression`1.xml]:
```suggestion
        <param name="visitor">The <see cref="T:System.Linq.Expressions.ExpressionVisitor"> with which this node will be visited.</param>
```

File: xml/System.Linq.Expressions/GotoExpression.xml [hunk 3]
```diff
@@ -81,7 +81,7 @@
       </Parameters>
       <Docs>
         <param name="visitor">To be added.</param>
```
Comment (carlossanlop) [inline xml/System.Linq.Expressions/GotoExpression.xml]:
```suggestion
        <param name="visitor">The <see cref="T:System.Linq.Expressions.ExpressionVisitor"> with which this node will be visited.</param>
```

File: xml/System.Linq.Expressions/IndexExpression.xml [hunk 4]
```diff
@@ -85,7 +85,7 @@
       </Parameters>
       <Docs>
         <param name="visitor">To be added.</param>
```
Comment (carlossanlop) [inline xml/System.Linq.Expressions/IndexExpression.xml]:
```suggestion
        <param name="visitor">The <see cref="T:System.Linq.Expressions.ExpressionVisitor"> with which this node will be visited.</param>
```

File: xml/System.Linq.Expressions/InvocationExpression.xml [hunk 5]
```diff
@@ -93,7 +93,7 @@
       </Parameters>
       <Docs>
         <param name="visitor">To be added.</param>
```
Comment (carlossanlop) [inline xml/System.Linq.Expressions/InvocationExpression.xml]:
```suggestion
         <param name="visitor">The <see cref="T:System.Linq.Expressions.ExpressionVisitor"> with which this node will be visited.</param>
```

File: xml/System.Linq.Expressions/LabelExpression.xml [hunk 6]
```diff
@@ -71,7 +71,7 @@
       </Parameters>
       <Docs>
         <param name="visitor">To be added.</param>
```
Comment (carlossanlop) [inline xml/System.Linq.Expressions/LabelExpression.xml]:
```suggestion
         <param name="visitor">The <see cref="T:System.Linq.Expressions.ExpressionVisitor"> with which this node will be visited.</param>
```

File: xml/System.Linq.Expressions/ListInitExpression.xml [hunk 7]
```diff
@@ -89,7 +89,7 @@
       </Parameters>
       <Docs>
         <param name="visitor">To be added.</param>
```
Comment (carlossanlop) [inline xml/System.Linq.Expressions/ListInitExpression.xml]:
```suggestion
         <param name="visitor">The <see cref="T:System.Linq.Expressions.ExpressionVisitor"> with which this node will be visited.</param>
```

File: xml/System.Linq.Expressions/LoopExpression.xml [hunk 8]
```diff
@@ -81,7 +81,7 @@
       </Parameters>
       <Docs>
         <param name="visitor">To be added.</param>
```
Comment (carlossanlop) [inline xml/System.Linq.Expressions/LoopExpression.xml]:
```suggestion
         <param name="visitor">The <see cref="T:System.Linq.Expressions.ExpressionVisitor"> with which this node will be visited.</param>
```

File: xml/System.Linq.Expressions/MemberInitExpression.xml [hunk 9]
```diff
@@ -89,7 +89,7 @@
       </Parameters>
       <Docs>
         <param name="visitor">To be added.</param>
```
Comment (carlossanlop) [inline xml/System.Linq.Expressions/MemberInitExpression.xml]:
```suggestion
         <param name="visitor">The <see cref="T:System.Linq.Expressions.ExpressionVisitor"> with which this node will be visited.</param>
```

File: xml/System.Linq.Expressions/RuntimeVariablesExpression.xml [hunk 10]
```diff
@@ -78,7 +78,7 @@
       </Parameters>
       <Docs>
         <param name="visitor">To be added.</param>
```
Comment (carlossanlop) [inline xml/System.Linq.Expressions/RuntimeVariablesExpression.xml]:
```suggestion
         <param name="visitor">The <see cref="T:System.Linq.Expressions.ExpressionVisitor"> with which this node will be visited.</param>
```

File: xml/System.Linq.Expressions/SwitchExpression.xml [hunk 11]
```diff
@@ -81,7 +81,7 @@
       </Parameters>
       <Docs>
         <param name="visitor">To be added.</param>
```
Comment (carlossanlop) [inline xml/System.Linq.Expressions/SwitchExpression.xml]:
```suggestion
         <param name="visitor">The <see cref="T:System.Linq.Expressions.ExpressionVisitor"> with which this node will be visited.</param>
```

File: xml/System.Linq.Expressions/TryExpression.xml [hunk 12]
```diff
@@ -96,7 +96,7 @@
       </Parameters>
       <Docs>
         <param name="visitor">To be added.</param>
```
Comment (carlossanlop) [inline xml/System.Linq.Expressions/TryExpression.xml]:
```suggestion
         <param name="visitor">The <see cref="T:System.Linq.Expressions.ExpressionVisitor"> with which this node will be visited.</param>
```

File: xml/System.Linq.Expressions/TypeBinaryExpression.xml [hunk 13]
```diff
@@ -91,7 +91,7 @@
       </Parameters>
       <Docs>
         <param name="visitor">To be added.</param>
```
Comment (carlossanlop) [inline xml/System.Linq.Expressions/TypeBinaryExpression.xml]:
```suggestion
         <param name="visitor">The <see cref="T:System.Linq.Expressions.ExpressionVisitor"> with which this node will be visited.</param>
```

File: xml/System.Linq.Expressions/UnaryExpression.xml [hunk 14]
```diff
@@ -101,7 +101,7 @@
       </Parameters>
       <Docs>
         <param name="visitor">To be added.</param>
```
Comment (carlossanlop) [inline xml/System.Linq.Expressions/UnaryExpression.xml]:
```suggestion
         <param name="visitor">The <see cref="T:System.Linq.Expressions.ExpressionVisitor"> with which this node will be visited.</param>
```

File: xml/System.Linq.Expressions/DefaultExpression.xml [hunk 15]
```diff
@@ -81,7 +81,7 @@
...
-        <summary>To be added.</summary>
+        <summary>Dispatches to the specific visit method for this node type.</summary>
         <returns>To be added.</returns>
```
Comment (carlossanlop) [inline xml/System.Linq.Expressions/DefaultExpression.xml]:
```suggestion
        <returns>The result of visiting this node.</returns>
```

File: xml/System.Linq.Expressions/Expression`1.xml [hunk 16]
```diff
@@ -101,7 +101,7 @@
...
-        <summary>To be added.</summary>
+        <summary>Dispatches to the specific visit method for this node type.</summary>
         <returns>To be added.</returns>
```
Comment (carlossanlop) [inline xml/System.Linq.Expressions/Expression`1.xml]:
```suggestion
         <returns>The result of visiting this node.</returns>
```

File: xml/System.Linq.Expressions/GotoExpression.xml [hunk 17]
```diff
@@ -81,7 +81,7 @@
...
-        <summary>To be added.</summary>
+        <summary>Dispatches to the specific visit method for this node type.</summary>
         <returns>To be added.</returns>
```
Comment (carlossanlop) [inline xml/System.Linq.Expressions/GotoExpression.xml]:
```suggestion
         <returns>The result of visiting this node.</returns>
```

File: xml/System.Linq.Expressions/IndexExpression.xml [hunk 18]
```diff
@@ -85,7 +85,7 @@
...
-        <summary>To be added.</summary>
+        <summary>Dispatches to the specific visit method for this node type.</summary>
         <returns>To be added.</returns>
```
Comment (carlossanlop) [inline xml/System.Linq.Expressions/IndexExpression.xml]:
```suggestion
         <returns>The result of visiting this node.</returns>
```

File: xml/System.Linq.Expressions/InvocationExpression.xml [hunk 19]
```diff
@@ -93,7 +93,7 @@
...
-        <summary>To be added.</summary>
+        <summary>Dispatches to the specific visit method for this node type.</summary>
         <returns>To be added.</returns>
```
Comment (carlossanlop) [inline xml/System.Linq.Expressions/InvocationExpression.xml]:
```suggestion
         <returns>The result of visiting this node.</returns>
```

File: xml/System.Linq.Expressions/LabelExpression.xml [hunk 20]
```diff
@@ -71,7 +71,7 @@
...
-        <summary>To be added.</summary>
+        <summary>Dispatches to the specific visit method for this node type.</summary>
         <returns>To be added.</returns>
```
Comment (carlossanlop) [inline xml/System.Linq.Expressions/LabelExpression.xml]:
```suggestion
         <returns>The result of visiting this node.</returns>
```

File: xml/System.Linq.Expressions/ListInitExpression.xml [hunk 21]
```diff
@@ -89,7 +89,7 @@
...
-        <summary>To be added.</summary>
+        <summary>Dispatches to the specific visit method for this node type.</summary>
         <returns>To be added.</returns>
```
Comment (carlossanlop) [inline xml/System.Linq.Expressions/ListInitExpression.xml]:
```suggestion
         <returns>The result of visiting this node.</returns>
```

File: xml/System.Linq.Expressions/LoopExpression.xml [hunk 22]
```diff
@@ -81,7 +81,7 @@
...
-        <summary>To be added.</summary>
+        <summary>Dispatches to the specific visit method for this node type.</summary>
         <returns>To be added.</returns>
```
Comment (carlossanlop) [inline xml/System.Linq.Expressions/LoopExpression.xml]:
```suggestion
         <returns>The result of visiting this node.</returns>
```

File: xml/System.Linq.Expressions/MemberInitExpression.xml [hunk 23]
```diff
@@ -89,7 +89,7 @@
...
-        <summary>To be added.</summary>
+        <summary>Dispatches to the specific visit method for this node type.</summary>
         <returns>To be added.</returns>
```
Comment (carlossanlop) [inline xml/System.Linq.Expressions/MemberInitExpression.xml]:
```suggestion
         <returns>The result of visiting this node.</returns>
```

File: xml/System.Linq.Expressions/RuntimeVariablesExpression.xml [hunk 24]
```diff
@@ -78,7 +78,7 @@
...
-        <summary>To be added.</summary>
+        <summary>Dispatches to the specific visit method for this node type.</summary>
         <returns>To be added.</returns>
```
Comment (carlossanlop) [inline xml/System.Linq.Expressions/RuntimeVariablesExpression.xml]:
```suggestion
         <returns>The result of visiting this node.</returns>
```

File: xml/System.Linq.Expressions/SwitchExpression.xml [hunk 25]
```diff
@@ -81,7 +81,7 @@
...
-        <summary>To be added.</summary>
+        <summary>Dispatches to the specific visit method for this node type.</summary>
         <returns>To be added.</returns>
```
Comment (carlossanlop) [inline xml/System.Linq.Expressions/SwitchExpression.xml]:
```suggestion
         <returns>The result of visiting this node.</returns>
```

File: xml/System.Linq.Expressions/TryExpression.xml [hunk 26]
```diff
@@ -96,7 +96,7 @@
...
-        <summary>To be added.</summary>
+        <summary>Dispatches to the specific visit method for this node type.</summary>
         <returns>To be added.</returns>
```
Comment (carlossanlop) [inline xml/System.Linq.Expressions/TryExpression.xml]:
```suggestion
         <returns>The result of visiting this node.</returns>
```

File: xml/System.Linq.Expressions/TypeBinaryExpression.xml [hunk 27]
```diff
@@ -91,7 +91,7 @@
...
-        <summary>To be added.</summary>
+        <summary>Dispatches to the specific visit method for this node type.</summary>
         <returns>To be added.</returns>
```
Comment (carlossanlop) [inline xml/System.Linq.Expressions/TypeBinaryExpression.xml]:
```suggestion
         <returns>The result of visiting this node.</returns>
```

File: xml/System.Linq.Expressions/UnaryExpression.xml [hunk 28]
```diff
@@ -101,7 +101,7 @@
...
-        <summary>To be added.</summary>
+        <summary>Dispatches to the specific visit method for this node type.</summary>
         <returns>To be added.</returns>
```
Comment (carlossanlop) [inline xml/System.Linq.Expressions/UnaryExpression.xml]:
```suggestion
         <returns>The result of visiting this node.</returns>
```

**Comment by carlossanlop** [comment]
@rpetrusha done.

**Comment by rpetrusha** [comment]
Thanks for making all of the additional changes, @carlossanlop. I'll merge your PR now.

**Comment by carlossanlop** [comment]
Thank you @rpetrusha !

**Pull request merged**